rootfs/standard/usr/share/mynode/application_info.json

```diff
@@ -319,5 +319,9 @@
         "short_name": "ufw",
         "show_on_application_page": false,
         "journalctl_log_name": "ufw"
+    },
+    {
+        "name": "Joinmarket GUI",
+        "short_name": "jmg"
     }
```

Comment:
In case there are issues, I think adding it as a "standalone" app that shows up on the application page may be better for users. Maybe follow a pattern similar to LNDConnect or Pool.